src/config.ts

```diff
@@ -21,20 +21,26 @@ try {
 
 export const yamlConfig = _.merge(defaultConfig, customConfig)
 
-export class TransactionLimits implements ITransactionLimits {
-  readonly config
-  readonly level
-
-  constructor({ level }) {
-    this.config = yamlConfig.limits
-    this.level = level
+export const getGenericLimits = (limitsConfig = yamlConfig.limits): GenericLimits => ({
```

Comment:
generic or default limits?

Reply:
![image](https://user-images.githubusercontent.com/17693119/127869673-0ec63578-8443-4754-8587-0f6b40c28159.png)

The whole set of limits are default limits, and then within that I've been thinking of modeling them as:
- `AccessLimits`
  Login attempts, phone code related limits etc.

- `UserLimits`
  Limits that depend on a `level`property

- `GenericLimits`
    Limits that _don't_ depend on any properties

- `TransactionLimits`
    `UserLimits` + `GenericLimits`

Reply:
so are they defaultTransactionLimits  ? @krtk6160 what do you think?

Reply:
Yeah default is kind of redundant here, I don't think we need to include `default` in the naming itself, because we don't reference any non-default values inside the code.

Reply:
@bodymindarts What do you think about @vindard's proposed approach of modelling the limits?

Reply:
uhmm so Generic limits are the global transaction limits? (not related to the user?)